etc/adminhtml/config.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Store:etc/config.xsd">
+    <default>
+        <selectivecache>
+            <general>
```

Comment:
It is not clear that these settings belong to cron. Consider renaming the section to cron